apps/src/StudioApp.js

```diff
@@ -1,19 +1,5 @@
 /* global Blockly, ace:true, droplet, marked, dashboard, addToHome */
 
-/**
- * For the most part, we depend on dashboard providing us with React as a global.
- * However, there is at least one context in which this won't be true - when we
- * show feedback blocks in their own iframe. In that case, load React from
- * our module.
- * This approach has a couple of drawbacks
- * (1) ability for dashboard React and apps React versions to get out of sync
- * (2) if we end up in other cases where React isn't provided to us as a global
- *     we probably won't notice, which may not be intended behavior
- */
-if (!window.React) {
-  window.React = require('react');
-}
-
```

Comment:
Turns out this was already unneeded, as feedbackBlocks stopped using iframes a little while back.